src/Collections/WorkflowLogsCollection.php

```diff
@@ -161,8 +161,8 @@ public function findLatestFromOptionsArray(
         if (
             $wfl && $wfl->get('start_time') >= $start_time &&
             (
-                $wfl->get('type') === $options['type'] ||
-                $wfl->get('commit_hash') === $options['commit_hash']
+                $wfl->get('type') === ($options['type'] ?? '') ||
+                $wfl->get('commit_hash') === ($options['commit_hash'] ?? '')
```

Comment:
The commit_hash clause here is probably ok, since there can be only one match; however, I think this use case is better served by the later code. Comparing by type here is likely harmful, though, as it might return the wrong workflow. I still think the easiest fix is to delete this entire if block, and rely on the code that follows.